src/main/java/frc/util/math/Trajectory.java

```diff
@@ -0,0 +1,75 @@
+package frc.util.math;
+
+
+public class Trajectory {
+    
+    // Create variables to store input data
+    private double distance;
+    private double height;
+    
+    // Set input distance
+    public void setDistance(double distance) {
+        this.distance = distance;
+    }
+    
+    // Set input Height
+    public void setHeight(double height) {
+        this.height = height;
+    }
+    
+    // Return angle after calculations
+    public double getAngle() {
+        return calculateAngle(this.distance, this.height);
+    }
+    
+    // Reutrn velocity after calculations
+    public double getVelocity() {
+        return calculateVelocity(this.distance, this.height);
+    }
+    
+    //call setShootingAngle to calculate for the nessessary shooter angle
+    private double calculateAngle(double distance, double height){
+
+        //calculate theoretical distance using given distance
+        double theoDistanceC1 = Math.pow(distance,2)*0.0378;
```

Comment:
Any numbers that are used for these calculations should be declared as private static final at the top of the code, especially if they are used multiple times. I don't know what 0.0378 means.

Reply:
Ok, agreed

Reply:
@CBTdungeon  Jake buddy, can you do this pls? I mean it's your algorithm.